docs/configuration/configuration.md

```diff
@@ -6,17 +6,18 @@ highest priority, followed by environment variables and finally config file.
 Supported flags are as follows (result of running `./flagd start --help`):
 
 ```
-  -b, --bearer-token string       Set a bearer token to use for remote sync
-  -e, --evaluator string          Set an evaluator e.g. json (default "json")
-  -h, --help                      help for start
-  -p, --port int32                Port to listen on (default 8013)
-  -m, --metrics-port int32        Port to set up metrics listener on (default 8014)
-  -c, --server-cert-path string   Server side tls certificate path
-  -k, --server-key-path string    Server side tls key path
-  -a, --sync-provider-args        Sync provider arguments as key values separated by =
-  -d, --socket-path string        Set the flagd socket path.
-  -f, --uri strings               Set a sync provider uri to read data from this can be a filepath, url or reference to a kubernetes custom resource. Using multiple providers is supported, however if flag keys are duplicated across multiple sources it may lead to unexpected behavior.
-  -C, --cors-origin strings       Set a CORS allow origin header, setting "*" will allow all origins (by default CORS headers are not set)
+  -b, --bearer-token string                 Set a bearer token to use for remote sync
+  -C, --cors-origin strings                 CORS allowed origins, * will allow all origins
+  -e, --evaluator string                    Set an evaluator e.g. json, yaml/yml (default "json")
+  -h, --help                                help for start
+  -m, --metrics-port int32                  Port to serve metrics on (default 8014)
+  -p, --port int32                          Port to listen on (default 8013)
+  -c, --server-cert-path string             Server side tls certificate path
+  -k, --server-key-path string              Server side tls key path
+  -d, --socket-path string                  Flagd socket path. With grpc the service will become available on this address. With http(s) the grpc-gateway proxy will use this address internally.
+  -y, --sync-provider string                DEPRECATED: Set a sync provider e.g. filepath or remote
+  -a, --sync-provider-args stringToString   Sync provider arguments as key values separated by = (default [])
+  -f, --uri strings                         Set a sync provider uri to read data from, this can be a filepath,url or FeatureFlagConfiguration. Using multiple providers is supported however ifflag keys are duplicated across multiple sources it may lead to unexpected behavior
```

Comment:
```suggestion
  -f, --uri strings                         Set a sync provider uri to read data from, this can be a filepath, url or FeatureFlagConfiguration. Using multiple providers is supported however if flag keys are duplicated across multiple sources it may lead to unexpected behavior.
```

Reply:
Looks there is a small [typo/bug in the existing code](https://github.com/open-feature/flagd/blob/8c8e155342154b4e1af87e24e68ab71a9446699e/cmd/start.go#L47-L48)

Reply:
Fixed. Thank you!

Reply:
Looks like these changes are still required

Reply:
I think I forgot to click `Sign off and commit suggestion` after writing the commit message. Fixed. Thank you for noticing it!